react-demo/src/Logo.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 
-type PXBlueSVGProps = {
+type BrightlayerUISVGProps = {
     className?: string;
     color?: string;
     size?: number;
 };
-export const PXBlueSVG: React.FC<PXBlueSVGProps> = (props) => {
+export const BrightlayerUISVG: React.FC<BrightlayerUISVGProps> = (props) => {
```

Comment:
This is still a PXBlue logo so we don't need to change the name here. It doesn't look like it's being used anywhere in the demo so I'm tempted to just rip it out entirely. Thoughts? @emclaug2 @joebochill

Reply:
For the sake of time, just revert the name change. We can rip it out later if we want.